### PR TITLE
PXB-3195 PXB 8.2.0-1 release notes and release tasks

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/release-notes/8.2.0-1.md
	modified:   docs/release-notes/release-notes.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.2.0-1.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Install [Percona XtraBackup 8.2](..//installation.md)
+
+The [Percona XtraBackup GitHub repository](https://github.com/percona/percona-xtrabackup)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
+
+[Percona XtraBackup 8.2]: https://docs.percona.com/percona-xtrabackup/inno-release/
+
+[PXB-3003]: https://jira.percona.com/browse/PXB-3003
```
Comment (satya-bodapati) [inline docs/release-notes/8.2.0-1.md:37]: why are these not in the above bug fixes sections with a description?

Comment (patrickbirch) [reply docs/release-notes/8.2.0-1.md:37]: Just a different way to add Markdown links.

File: docs/release-notes/8.2.0-1.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+[PXB-3174]: https://jira.percona.com/browse/PXB-3174
+[PXB-3181]: https://jira.percona.com/browse/PXB-3181
+[PXB-3173]: https://jira.percona.com/browse/PXB-3173
```
Comment (satya-bodapati) [inline docs/release-notes/8.2.0-1.md:41]: 3173 and 3181 are mismatching?

Comment (satya-bodapati) [reply docs/release-notes/8.2.0-1.md:41]: nope, only 3173 is missing

File: docs/release-notes/8.2.0-1.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+The [Percona XtraBackup GitHub repository](https://github.com/percona/percona-xtrabackup)
+
+Download product binaries, packages, and tarballs at [Percona Product Downloads](https://www.percona.com/downloads)
+
+[Percona XtraBackup 8.2]: https://docs.percona.com/percona-xtrabackup/inno-release/
+
+[PXB-3003]: https://jira.percona.com/browse/PXB-3003
+[PXB-3147]: https://jira.percona.com/browse/PXB-3147
+[PXB-3147]: https://jira.percona.com/browse/PXB-3147
```
Comment (satya-bodapati) [inline docs/release-notes/8.2.0-1.md]: 3147 is duplicated. Now this explains why the count doesn't match :-) May be the old style is better ;)

File: docs/release-notes/8.2.0-1.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Bug fixes
+
+* [PXB-3003]: Percona XtraBackup discovering redo logs to parse and the server purging redo logs at the same time could cause a race condition.
+
+* [PXB-3147]: XtraBackup failed to execute the 'DO innodb_redo_log_consumer_register("PXB");' query.
+
+* [PXB-3168]: Under a high write load, the backup fails with "log block numbers mismatch" error.
+
+* [PXB-3173]: Partial download functionality regression
```
Comment (altmannmarcelo) [inline docs/release-notes/8.2.0-1.md]: Can we make this consistent with 8.0.35 RN ?

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**